readme.md

```diff
@@ -74,9 +60,13 @@ npm install
 npm link
 ```
 
-- If you want to use an unreleased version of qxcompiler, download it and 
+- If you want to use an unreleased version of qxcompiler, download it with `git clone https://github.com/qooxdoo/qooxdoo-compiler` and 
   `npm link path/to/qxcompiler` from the `qx-cli` directory.
 
+- If you want to use an unreleased version of qooxdoo, download it with `git clone https://github.com/qooxdoo/qooxdoo` and 
+  `npm link path/to/qooxdoo` from the `qx-cli` directory.
+
+- Pay attention: the linking must be repeated after an `npm install`. `npm install` substitutes the links with the downloaded version.
```

Comment:
Maybe better: "`npm install` substitutes the links with the NPM modules" (since both unreleased and release versions are "downloaded")